oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -493,4 +496,5 @@ Component-Server-Mappings:
   Mysqli: Mysql
   influxdb-java: InfluxDB
   Predis: Redis
-  PyMysql: Mysql
\ No newline at end of file
+  PyMysql: Mysql
+  HBase: HBase
```

Comment:
Whether we don't need this? Have you checked locally?

Reply:
fixed